From 89fb55d22f64eda26d18c735dfd59ee99b5191c2 Mon Sep 17 00:00:00 2001
From: Vendula Poncova <vponcova@redhat.com>
Date: Tue, 23 Feb 2021 15:50:50 +0100
Subject: [PATCH] The network spoke should be visible in live spins (#1932961)

It should be always possible to configure the hostname for the target system
in the graphical network spoke, unless the spoke was specifically disabled in
the Anaconda configuration files.

This regression was caused by the commit 6b80b7b.

Resolves: rhbz#1932961
---
 pyanaconda/ui/gui/spokes/network.py | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/pyanaconda/ui/gui/spokes/network.py b/pyanaconda/ui/gui/spokes/network.py
index 1ef468a084..3cae5b4489 100644
--- a/pyanaconda/ui/gui/spokes/network.py
+++ b/pyanaconda/ui/gui/spokes/network.py
@@ -1472,7 +1472,8 @@ class NetworkSpoke(FirstbootSpokeMixIn, NormalSpoke):
         if not FirstbootSpokeMixIn.should_run(environment, data):
             return False
 
-        return conf.system.can_configure_network
+        # Always allow to configure the hostname for the target system.
+        return True
 
     def __init__(self, *args, **kwargs):
         NormalSpoke.__init__(self, *args, **kwargs)
-- 
2.26.2

